examples/react-server/vite.config.ts

```diff
@@ -226,7 +226,7 @@ function vitePluginUseClient(): PluginOption {
       }
       manager.clientReferenceMap.delete(id);
       if (code.includes("use client")) {
-        const runtimeId = await normalizeReferenceId(id, "react-server");
+        const runtimeId = await normalizeReferenceId(id, "client");
```

Comment:
It looks like this was a simple typo, but I'm not sure then what's happening in https://github.com/hi-ogawa/vite-plugins/pull/297